packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -145,3 +147,10 @@ export interface CustomTagEnhancers {
 
 export type DefaultProps = ComponentDefaultPropsGeneric<CookieBannerProps, 'hasPrimaryActionsOnly' | 'defaultPreferences' | 'locale' | 'cookieStatementLink' | 'cookieTechnologiesLink'>;
```

Comment:
After #1582 was merged you should now able to do this:
```suggestion
export type DefaultProps = ComponentDefaultProps<CookieBannerProps>;
```

Reply:
I just updated so this can get merged 😄